book/content/collaborating_github/collaborating_github.md

```diff
@@ -2,7 +2,7 @@
 
 
 ## Prerequisites/recommended skill level
-As both GitHub and GitLab are based on git, we recommend you familiarize yourself with version control before continuing. 
+As both GitHub and GitLab are based on git, we recommend you familiarize yourself with [version control](/version_control/version_control.html) before continuing. 
```

Comment:
So sorry to be weirdly picky about this. For consistency, can we drop the `.html`? I've suggested the change below:

```suggestion
As both GitHub and GitLab are based on git, we recommend you familiarize yourself with [version control](/version_control/version_control) before continuing. 
```

Reply:
Actually, this should be a table, right? Like the open_research chapter below?